src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILocalReferenceExpression.cs

```diff
@@ -30,7 +30,10 @@ public void M2(out int i )
 }
 ";
             string expectedOperationTree = @"
-ILocalReferenceExpression: i (IsDeclaration: True) (OperationKind.LocalReferenceExpression, Type: System.Int32) (Syntax: 'var i')
+IArgument (ArgumentKind.Explicit, Matching Parameter: i) (OperationKind.Argument) (Syntax: 'var i')
```

Comment:
This seems to be affecting large number of tests. Can we fix the factory to not return IArgument for syntax kinds other than argument?

Reply:
sure I can take a look at argument.